.github/workflows/ci.yml

```diff
@@ -45,13 +45,32 @@ jobs:
         with:
           python-version: "3.7.x"
           architecture: "x64"
-      - run: pip install pydocstyle
-      - run: pydocstyle --convention=numpy --add-ignore=D101,D102,D105 modin/pandas/dataframe.py modin/pandas/series.py modin/pandas/groupby.py modin/pandas/base.py
-      - run: pydocstyle --convention=numpy --add-ignore=D101,D103,D105 modin/pandas/io.py
-      - run: pydocstyle --convention=numpy --add-ignore=D101,D102 modin/pandas/series_utils.py
-      - run: pydocstyle --convention=numpy --add-ignore=D103 modin/pandas/general.py
-      - run: pydocstyle --convention=numpy modin/pandas/plotting.py modin/pandas/utils.py modin/pandas/iterator.py modin/pandas/indexing.py
-      - run: pydocstyle --convention=numpy --add-ignore=D100,D104 modin/engines/base/frame
+      - run: pip install pydocstyle numpydoc
+      - run: pip install -e .[all]
+      - run: |
+          python scripts/doc-checker.py --add-ignore=D101,D102,D105 --disable-numpydoc \
```

Comment:
Why do we disable numpydoc for modin/pandas?

Reply:
I disabled this check only for files where docstrings from pandas are inherited, because:
1) There is no point in checking someone else's documentation;
2) Our checks are different from pandas checks;

Reply:
But this leaves out checks for non-inherited methods, which doesn't feel right.

Reply:
Ok, in this case monkeypatching will also help. 

But the work on the documentation also gave me the idea that it is possible to leave modules in which only public functions are defined and docstrings are inherited, and all internal logic can be transferred either to the base class or to utilities.

Reply:
That is refactoring, not documenting. :) Different things.

Reply:
@YarShev now check is disabled only where there is internal undocumented functionality.